README.md

```diff
@@ -1,8 +1,103 @@
 # zinger
 
-`zinger` is a connector between Kafka topics and Zed lakes.
-It can run in either direction: syncing a Zed lake to a Kafka topic or
-syncing a Kafka topic to a Zed lake.
+`zinger` is a connector between [Kafka](https://kafka.apache.org/)
+topics and
+[Zed lakes](https://github.com/brimdata/zed/tree/main/docs/lake).
+It can run in either direction: syncing a Zed data pool to a Kafka topic or
+syncing a Kafka topic to a Zed pool.
+
+## Installation
+
+To install `zinger`, clone this repo and run `make install`:
+```
+git clone https://github.com/brimdata/zinger.git
+cd zinger
+make install
+```
+Make sure you have Go installed in your environment and that GOPATH is
+in your shell path.
+
+## Quick Start
+
+For built-in help, run
+```
+zinger -h
+```
+Make sure your config files are setup for the Kafka cluster
+and schema registry (see below), then run some tests.
+
+List schemas in the registry:
+```
+zinger ls
+```
+Create a topic called `MyTopic` with one partition using your Kafka admin tools,
+then post some data to a topic:
+```
+echo '{s:"hello,world"}' | zinger produce -topic MyTopic -
+```
+See the record you created:
+```
+zinger consume -topic MyTopic
+```
+> Hit Ctrl-C to interrupt `zinger consume` as it will wait indefinitely
+> for data to arrive on the topic.
+
+In another shell, run a Zed lake service:
+```
+mkdir scratch
+cd scratch
+zed lake serve
+```
+Now, sync data from Kafka to a Zed lake:
+```
+zapi create -orderby kafka.offset:desc PoolA
+zinger sync from -topic MyTopic -pool PoolA
+```
+See the data in the Zed pool:
+```
+zapi query "from PoolA"
+```
+Next, create a topic called `MyTarget` with one partition using your Kafka admin tools,
+sync data from a Zed pool back to Kafka, and check that it made it:
+```
+zinger sync to -topic MyTarget -pool PoolA
+zinger consume -topic MyTarget
+```
+Finally, try out shaping.  Put a Zed script in `shape.zed`, e.g.,
+```
+cat shape.zed
+
+mymeta.foo:=string(2*(kafka.offset+1))
+```
+And shape the record from `MyTopic` into a new `PoolB`:
+```
+zapi create -orderby kafka.offset:desc PoolB
+zinger sync from -topic MyTopic -pool PoolB -shaper shape.zed
+zapi query "from PoolB"
+```
+
+## Configuration
+
+To configure `zinger` to talk to a Kafka cluster and a schema registry,
+you must create two files in `$HOME/.zinger`:
+[`kafka_config.json`](kafka_config.json) and
+[`schema_registry.json`](schema_registry.json).
```

Comment:
Maybe change `kafka_config.json` to `kafka.json` for consistency with `schema_registry.json`?
```suggestion
[`kafka_config.json`](kafka_config.json) and
[`schema_registry.json`](schema_registry.json).
```